pyproject.toml

```diff
@@ -40,11 +40,11 @@ requires = ["poetry-core>=1.3.2"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.pytest.ini_options]
-base_url = "https://localhost"
+base_url = "http://localhost"
```

Comment:
do we still need these settings? we use the live_server url or directly set the url in the deployed test. And for the deployed test we use the env variables from the .env file to start the app, right?

Reply:
Probably not. I'll test and remove it if it's not needed anymore.

Reply:
Yup, not needed anymore.